src/commands/community/nft/add.ts

```diff
@@ -35,6 +35,12 @@ async function executeNftAddCommand(args: string[], msg: Message) {
   const errorMessageCollection = dataCollection.error
   const dataChain = await respChain.json()
   switch (respCollection.status) {
+    case 400:
+      return buildDiscordMessage(
+        msg,
+        "NFT",
+        "Already added. Nft is in sync progress"
+      )
     case 200:
       return buildDiscordMessage(msg, "NFT", "Successfully add new collection")
```

Comment:
return buildDiscordMessage(msg, "NFT", "Successfully add new collection to queue")